spec/models/spree_avatax/shared_spec.rb

```diff
@@ -11,23 +11,24 @@
 
     context 'when the order is taxable' do
       let(:order) { create(:shipped_order, line_items_count: 1) }
-      it { should be true }
+      it { is_expected.to be true }
     end
 
     context 'when the order has no line items' do
       let(:order) { create(:order) }
-      it { should be false }
+      it { is_expected.to be false }
     end
 
     context 'when the order has no ship address' do
-      let(:order) { create(:shipped_order, line_items_count: 1, ship_address: nil) }
-      it { should be false }
+      let(:order) { create(:shipped_order, line_items_count: 1) }
+      before { order.ship_address = nil }
```

Comment:
was this the one that was broken? if we have to do this separately could we use `update_attributes!` instead of just setting the value in memory?  i'd feel more confident in that.